packages/mc-html-template/index.js

```diff
@@ -12,7 +12,7 @@ module.exports = templateParams => {
 
   const cssChunks = cssVendorChunks.concat(cssAppChunks).map(fileName => {
     const chunkPath = fileName.replace(/^\//, '');
-    return `<link href="__CDN_URL__${chunkPath}" rel='stylesheet' type='text/css'></link>`;
+    return `<link href="__CDN_URL__${chunkPath}" rel='stylesheet' type='text/css'>`;
```

Comment:
this doesn't give a warning in IE

Reply:
https://stackoverflow.com/questions/9943981/can-self-closing-link-tags-be-problematic